<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LockerWidget</class>
 <widget class="QWidget" name="LockerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="LockerButton" name="lockerButton_top">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>89</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_top" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>39</y>
     <width>303</width>
     <height>293</height>
    </rect>
   </property>
   <widget class="QOpenGLWidget" name="openGLWidget_top">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="LockerButton" name="lockerButton_front">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>338</y>
     <width>89</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_front" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>368</y>
     <width>303</width>
     <height>294</height>
    </rect>
   </property>
   <widget class="QOpenGLWidget" name="openGLWidget_front">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="LockerButton" name="lockerButton_side">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>668</y>
     <width>89</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_side" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>698</y>
     <width>303</width>
     <height>293</height>
    </rect>
   </property>
   <widget class="QOpenGLWidget" name="openGLWidget_side">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LockerButton</class>
   <extends>QPushButton</extends>
   <header>lockerbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
